share/instruments/update_instruments_xml.py

```diff
@@ -4,6 +4,25 @@
 def eprint(*args, **kwargs):
     print(*args, **kwargs, file=sys.stderr)
 
+if __name__ == '__main__' and sys.prefix == sys.base_prefix:
```

Comment:
I assume we do this only in this file because this is the only file that uses a non-pre-installed package (requests), is that right?

Reply:
There are a few other Python files using modules not in the standard library (requests and [markdown](https://pypi.org/project/Markdown/)).

```
src/framework/draw/thirdparty/freetype/freetype-2.13.1/tests/scripts/download-test-fonts.py
buildscripts/ci/release/correct_release_info.py
buildscripts/ci/learn/make_youtube_playlist_info.py
```

Now that the method has been approved in principle, I'll add it to the buildscripts too.

I won't add it to the one in freetype since that's thirdparty.